test/v2/fixtures/retrieveBalanceAndTransactions/FinancialDetailsItemFixture.scala

```diff
@@ -67,7 +67,7 @@ trait FinancialDetailsItemFixture {
   val financialDetailsItemMtdJson: JsValue =
     Json.parse("""
       |{
-      |  "subItem": "subItem",
+      |  "itemId": "subItem",
       |  "dueDate": "2022-02-02",
       |  "amount": 1.23,
       |  "clearingDate": "2021-01-01",
```

Comment:
Line 94 - can we assign "subItem" the value "001" to match the spec? According to[ the spec](https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?pageId=367723051) subItem is a string that should not exceed 3 characters

Reply:
I have updated it now. Thank you